setup/scripts/prom_node_exporter_install.sh

```diff
@@ -4,7 +4,7 @@
 
 cd ~ || exit
 echo "[INFO] Downloading prometheus onode exporter"
-wget https://github.com/prometheus/node_exporter/releases/download/v0.16.0/node_exporter-0.16.0.linux-amd64.tar.gz
+wget https://github.com/prometheus/node_exporter/releases/download/v0.17.0/node_exporter-0.17.0.linux-amd64.tar.gz
```

Comment:
perhaps these versions might be better declared in separate variables just for clarity?

Reply:
Good idea, I've reflected this across all the scripts, except for the zabbix install one